Makefile

```diff
@@ -199,7 +199,7 @@ pao-functests: cluster-label-worker-cnf pao-functests-only
 pao-functests-only:
 	@echo "Cluster Version"
 	hack/show-cluster-version.sh
-	hack/run-functests.sh
+	hack/run-test.sh -t "test/e2e/performanceprofile/functests" -p "--v -r --fail-fast --skip-package='5_latency_testing,2_performance_update' --flake-attempts=2 --junit-report=report.xml" -m "Running Functional Tests"
```

Comment:
I'm not super excited about default parameters in general, because I think they obscure the way a function/script works and could lead to hard to find errors ... 
I usually prefer explicit over implicit.

Reply:
@Tal-or and I have slightly different pov here, any other thought about this? @yanirq @ffromani